### ISSUE TITLE
Dependencies for each sample

### ISSUE THREAD

**Issue by JosephIbe** [hokein/electron-sample-apps#60]
Can someone tell me where to find the dependencies for each sample as package.json is missing that.

**Comment by KorbsStudio** [comment]
Good point, of course to state the obvious they all do require a version Electron.
For Electron, I recommend version v9.4.4 and up(v10, v11, and v12).
